tfjs-backend-wasm/src/cc/backend.cc

```diff
@@ -48,6 +48,7 @@ TensorInfo &get_tensor_info_out(const size_t tensor_id) {
 }
 
 size_t xnn_operator_count = 0;
+pthreadpool *threadpool = pthreadpool_create(4);
```

Comment:
It is best to not hardcode the number of threads, but get it dynamically from `emscripten_num_logical_cores`. I suggest to use `std::min(emscripten_num_logical_cores(), 4)`.